### PR TITLE
feat(remote-development): Add netman.nvim

### PR DESCRIPTION
**Pull request by Uzaaft**

### CODE REVIEW

**Comment by luxus** [comment]
does netman make sense without intigrating it into neo-tree?

**Comment by Uzaaft** [comment]
Yes, it's mentioned in the docs.

**Review by mehalter — CHANGES_REQUESTED** [review]
We should also configure the neo-tree integration here as well if neo-tree exists using the `optional` key.

File: lua/astrocommunity/remote-development/netman-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,4 @@
+return {
+  "miversen33/netman.nvim",
+  opts = {},
+}
```
Comment (mehalter) [inline lua/astrocommunity/remote-development/netman-nvim/init.lua:4]:
```suggestion
return {
  {
    "miversen33/netman.nvim",
    cmd = {
      "NmloadProvider",
      "Nmlogs",
      "Nmdelete",
      "Nmread",
      "Nmwrite",
    },
    opts = {},
  },
  {
    "nvim-neo-tree/neo-tree.nvim",
    dependencies = { "netman.nvim" },
    optional = true,
    opts = function(_, opts)
      opts.sources = require("astronvim.utils").list_insert_unique(opts.sources, "netman.ui.neo-tree")
    end,
  },
}
```

**Review by mehalter — CHANGES_REQUESTED** [review]

**Review by mehalter — APPROVED** [review]
LGTM!

**Pull request merged**